tests/Rector/Property/NetteInjectToConstructorInjectionRector/Fixture/parent_with_constructor.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+use Nette\Security\Passwords;
+
+class TestPresenter extends BasePresenter
+{
+    /**
+     * @inject
+     */
+    public \Nette\Security\Passwords $passwords;
+
+    public function actionTest()
+    {
+        $this->passwords->hash('test');
+    }
+}
+
+abstract class BasePresenter
+{
+    public function __construct()
+    {
+    }
+}
+?>
+-----
+<?php
+use Nette\Security\Passwords;
+
+class TestPresenter extends BasePresenter
+{
+    public function __construct(private \Nette\Security\Passwords $passwords)
+    {
+        parent::__construct();
+    }
+
+    public function actionTest()
+    {
+        $this->passwords->hash('test');
+    }
+}
+
+abstract class BasePresenter
+{
+    public function __construct()
+    {
+    }
+}
+?>
```

Comment:
The extra presenter class that is not changed, but used for static analysis, should be located in `/Source` directory.